test/Confluent.Kafka.IntegrationTests/Tests/Consumer_Background.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2016-2017 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Xunit;
+
+
+namespace Confluent.Kafka.IntegrationTests
+{
+    public static partial class Tests
+    {
+        /// <summary>
+        ///     Basic Consumer test (background poll mode).
+        /// </summary>
+        [Theory, MemberData(nameof(KafkaParameters))]
+        public static void Consumer_Background(string bootstrapServers, string topic)
```

Comment:
Agree with all the suggestions about renaming tests but I'm going to leave as is in this PR and do a test restructure at a later time (priority right now is to get the API locked down).